test/services.ts

```diff
@@ -99,7 +99,7 @@ const createService = async <T>(
 ): Promise<ServiceDescription> => {
   const server = new ApolloServer({
     schema: buildSubgraphSchema({
-      resolvers,
+      resolvers: resolvers as GraphQLResolverMap<unknown>,
```

Comment:
why do we need this cast?

Reply:
Because of the Generic used on `resolvers`
- `buildSubgraphSchema` does not infer from the expect a generic anymore. It when receiving a `GraphQLResolverMap`, it expects it to be a `GraphQLResolverMap<unknown>`.

Reply:
exactly

Reply:
yep, without this cast TS will complain 

<img width="622" alt="image" src="https://github.com/profusion/apollo-federation-node-gateway/assets/17731880/6a2a2578-733b-45b9-8304-9a677503f2b0">